### PR TITLE
request initial values when polling/webhook starts

### PR DESCRIPTION
**Pull request by mbanders**
To be more user-friendly, we request initial resource values from
mbed cloud. That way we can populate charts and such right away.

Signed-off-by: mbanders <michael.anderson@arm.com>

### CODE REVIEW

File: livedevice/mbedcloud.py [hunk 1]
```diff
@@ -56,8 +56,11 @@ def get_endpoint_resource(self, endpoint_id, resource_path):
         if not (resource_path and resource_path.startswith('/')):
             return
         r = self.get("/v2/endpoints/%s%s" % (endpoint_id, resource_path))
-        r.raise_for_status()
-        return r.json()
+        try:
+            return r.json()
+        except ValueError as e:
+            print("Warn: %s does not have %s" % (endpoint_id, resource_path))
```
[the remark this answers is not in the document]
Comment (tubaman) [reply livedevice/mbedcloud.py]: I don't like prints in the mbedcloud API wrapper,  I'd rather let the exception bubble up to the code above.  Are you not getting a 404 when an endpoint doesn't have a resource?

File: livedevice/models.py [hunk 2]
```diff
@@ -188,8 +189,22 @@ def set_webhook_callback(self):
...
+    def request_initial_resource_values(self):
+        '''Request values for resource paths to obtain initial values.'''
+        session = self.get_session()
+        endpoints = session.get_endpoints()
+        for e in endpoints:
+            endpoint_id = e['name']
+            for p in settings.RESOURCE_PATHS_FOR_INIT:
+                result = session.get_endpoint_resource(endpoint_id, p)
+                if 'async-response-id' in result:
+                    print("Seting cache for %s " % result['async-response-id'])
```
[the remark this answers is not in the document]
Comment (tubaman) [reply livedevice/models.py]: Let's change that print to `logger.debug`

File: livedevice/models.py [hunk 3]
```diff
@@ -188,8 +189,22 @@ def set_webhook_callback(self):
...
+        self.request_initial_resource_values()
         self.set_presubscriptions()
 
+    def request_initial_resource_values(self):
+        '''Request values for resource paths to obtain initial values.'''
+        session = self.get_session()
+        endpoints = session.get_endpoints()
+        for e in endpoints:
+            endpoint_id = e['name']
+            for p in settings.RESOURCE_PATHS_FOR_INIT:
```
Comment (tubaman) [inline livedevice/models.py]: Is there a commit that adds this setting?

Comment (mbanders) [reply livedevice/models.py]: That was added: https://github.com/ARMmbed/ref-wem-webapp/commit/b1dd4f994ae78abbeeb90371ddd614221b856cc6

File: livedevice/models.py [hunk 4]
```diff
@@ -212,9 +227,10 @@ def start_long_polling(self):
         if self.is_long_polling:
             raise Exception("is already running")
         session = self.get_session()
-        session.delete_long_poll()
+        #session.delete_long_poll()
```
[the remark this answers is not in the document]
Comment (tubaman) [reply livedevice/models.py]: Can you squash this into the previous commit?

**Pull request merged**